ansible/group_vars/hypervisors

```diff
@@ -0,0 +1,38 @@
+---
+# Map physical network names to their source device. This can be either an
+# existing interface or an existing bridge.
+physnet_mappings: {}
+
+virtualisation_provider: libvirt
+
+system_requirements:
+  - python-pip
```

Comment:
Do you need to yum install `epel-release` to get this? It's possible to manage without it, just using pip in virtualenvs.

Reply:
Yeah, I think I did install `epel-release` beforehand. I'm only using pip in virtualenvs here - can I use the Ansible pip module with the `virtualenv` parameter if I don't install python-pip here?

Reply:
Yes, it should work.